prediction_market_agent_tooling/markets/seer/seer.py

```
@@ -281,14 +295,57 @@ def place_bet(
+        outcome: bool,
+        amount: TokenAmount,
+        auto_withdraw: bool = True,
+        api_keys: APIKeys | None = None,
+        web3: Web3 | None = None,
+    ) -> str:
+        """
+        Sells the given number of shares for the given outcome in the given market.
+        """
+        outcome_token = self.get_wrapped_token_for_outcome(outcome)
+        api_keys = api_keys if api_keys is not None else APIKeys()
+        amount_to_trade = xdai_type(amount.amount)
```

Ruff (0.8.2)

Undefined name `xdai_type`

(F821)